benchmarks/README.md

```diff
@@ -0,0 +1,193 @@
+
+### BenchmarkSettings
+
+```mermaid
+classDiagram
+    class BenchmarkSettings {
+        <<ABC, BenchmarkSerialization>>
+        +int random_seed
+    }
+```
+
+The settings object is used to parameterize the benchmark. It is an abstract base class that can be extended by the user to provide additional information for the benchmark. The only required attribute is `random_seed`, which is used to seed the entire call of the benchmark function. A subtype of the class can be used to provide more information to the benchmark function and can be added as needed for a specific benchmarking task. Additional setting types can be added to reflect parameterization for the benchmarking callable. Currently, the following settings are available:
+
+#### ConvergenceExperimentSettings
+
+```mermaid
+classDiagram
+    class ConvergenceExperimentSettings {
+        +int batch_size
+        +int n_doe_iterations
+        +int n_mc_iterations
+    }
+```
+
+The settings object is used to parameterize the convergence experiment benchmarks and holds information about the batch size, the number of design of experiment iterations, and the number of Monte Carlo iterations, which can be used for BayBE scenario executions. Please refer to the BayBE documentation for more information about experiment simulations.
```

Comment:
- "Please refer to the BayBE documentation for more information about experiment simulations." Either have an explicit link here or just delete.